### ISSUE TITLE
about results

### ISSUE THREAD

**Issue by BrownSweater** [clovaai/CRAFT-pytorch#37]
I use Synthtext to train CRAFT, but the result is very bad.
this is mine
![image](https://user-images.githubusercontent.com/35992872/63858720-97e50b80-c9d8-11e9-9d72-ecf40ce741b1.png)
![image](https://user-images.githubusercontent.com/35992872/63858731-9b789280-c9d8-11e9-87ec-19f48f7b7466.png)

this is yours
![image](https://user-images.githubusercontent.com/35992872/63858754-a4696400-c9d8-11e9-9461-614f08576e40.png)
![image](https://user-images.githubusercontent.com/35992872/63858754-a4696400-c9d8-11e9-9461-614f08576e40.png)

I don't know what caused this problem.I guess the difference is caused by loss, i use ohem, all postive (gt > 0.1) and negative(gt<0.1), pos:neg=1:3，and loss divede by number of neg and pos. 
this is train loss, Its value looks smaller. Can you give me some guidance?
![image](https://user-images.githubusercontent.com/35992872/63859171-67ea3800-c9d9-11e9-9efb-41dff50db137.png)

**Comment by mayank-git-hub** [comment]
I think the Gaussian heat map you are using has a very small standard deviation because of which the characters are not getting connected by the affinity. Try increasing the standard deviation and retrain the model and the problem should be solved. 

I don't think it is a problem caused by OHEM.

The loss values seem to be in the correct range.

**Comment by BrownSweater** [comment]
> I think the Gaussian heat map you are using has a very small standard deviation because of which the characters are not getting connected by the affinity. Try increasing the standard deviation and retrain the model and the problem should be solved.
> 
> I don't think it is a problem caused by OHEM.
> 
> The loss values seem to be in the correct range.

thanks for your answer , i increasing my variance to 3  in my gt,  it looks like better.
What is the good value in your experience?

**Comment by brooklyn1900** [comment]
May I ask, what do you mean by "postive (gt > 0.1) and negative(gt<0.1)"? 
Are they derived from gaussian map label or the region scores passed through the net?

Thanks  a lot!

**Comment by backtime92** [comment]
@mayank-git-hub i think 1.5-1.7 both ok, but i do not found a suitable value.

**Comment by backtime92** [comment]
@BrownSweater  i think 1.5-1.7 both ok, but i do not found a suitable value.

**Comment by waeliFatima** [comment]
I use the CRAFT and i have question about the result ..
In the res_craft_example.txt i not get the text of image it is contain just some number .
please tell me how to solve that error?
